<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectSetupWidget</class>
 <widget class="QWidget" name="ProjectSetupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProjectSetupWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Project setup</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <item row="2" column="0" colspan="3">
       <widget class="QLabel" name="p_lbl_ClosingCorporation">
        <property name="text">
         <string>Time of closing the work of the organization</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QTimeEdit" name="p_timeEdit_MinRestTime">
        <property name="dateTime">
         <datetime>
          <hour>10</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QTimeEdit" name="p_timeEdit_StandartRestTime">
        <property name="dateTime">
         <datetime>
          <hour>12</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="p_checkBox_RoundTheClockMode">
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QCheckBox" name="p_checkBox_EveryMounthGraphic">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QTimeEdit" name="p_timeEdit_ClosingCorporation">
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>22</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QTimeEdit" name="p_timeEdit_OpeningCorporation">
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>8</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QLabel" name="p_lbl_EveryMounthGraphic">
        <property name="text">
         <string>Creating a work schedule every month</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QLabel" name="p_lbl_MinRestTime">
        <property name="text">
         <string>Minimum rest time for workers</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QLabel" name="p_lbl_StandartRestTime">
        <property name="text">
         <string>Standart rest time for workers</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLabel" name="p_lbl_GraphicForeverDay">
        <property name="text">
         <string>Schedule every n days</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="p_lbl_OpeningCorporation">
        <property name="text">
         <string>Time of the beginning of work of the organization</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QSpinBox" name="p_spinBox_GraphicForeverDay">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>90</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="4">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="p_lbl_RoundTheClockMode">
        <property name="text">
         <string>Round-the-clock mode</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="p_buttons_layout">
     <item>
      <widget class="QPushButton" name="p_button_Open">
       <property name="text">
        <string>Open...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="p_button_Save">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="p_button_SaveAs">
       <property name="text">
        <string>Save as...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>p_checkBox_RoundTheClockMode</tabstop>
  <tabstop>p_timeEdit_OpeningCorporation</tabstop>
  <tabstop>p_timeEdit_ClosingCorporation</tabstop>
  <tabstop>p_checkBox_EveryMounthGraphic</tabstop>
  <tabstop>p_spinBox_GraphicForeverDay</tabstop>
  <tabstop>p_timeEdit_MinRestTime</tabstop>
  <tabstop>p_timeEdit_StandartRestTime</tabstop>
  <tabstop>p_button_Open</tabstop>
  <tabstop>p_button_Save</tabstop>
  <tabstop>p_button_SaveAs</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
